### PR TITLE
Plagiarism-Checker

### LINKED ISSUE #655: Plagiarism checker in Python
# Aim

- The objective of the script is to detect plagiarism in textual document using python.
# Details
This is a plagiarism checker made in python.It will compare the text files and tell if the document is copied or not.

# Features
 - Handling quotations
 - Automating the collection of training data.
 -  Comparing different submissions  to each other.
 -  Adapting this to work for plagiarized text.
 -  Checking if the text document is copied from google source.

### PR DESCRIPTION
**Pull request by Anupreetadas**
# Description

I have added the plagiarism checker project which helps user to check similarity between two text files.It computes similarity when the textual raw data is transformed into vectors and then to arrays of numbers and then from that we can compute the the similarity between them.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/Anupreetadas/Amazing-Python-Scripts/blob/branch_name2/Plagiarism-Checker/README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Alarm clock files are pushed here?

**Comment by Anupreetadas** [comment]
> Alarm clock files are pushed here?

I am so sorry ,It happened because I made another branch to push this project.It is fixed now . Sorry for the inconvinience caused.It wont happen again.

File: Plagiarism-Checker/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kaustubhgupta) [inline Plagiarism-Checker/.vscode/settings.json:1]: this file is not needed, you can delete it

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by antrikshmisri — APPROVED** [review]
LGTM 👍🏻

**Review by santushtisharma10 — APPROVED** [review]

**Comment by kaustubhgupta** [comment]
@Anupreetadas Umm, why you messed up this branch? Why there are alarm clock files in this branch?

**Pull request merged**